index.html

```diff
@@ -52,6 +52,11 @@
                         </a>
                     </button>
                 </li>
+                <li class="list-inline-item">
+                    <a href="#" class="nav-icon-link link-style-none">
+                        <i class="fas fa-sun nav-icon dark"></i>
```

Comment:
reset the code ! rest everything is clean

Reply:
Ok, I will check. Thank you for review